hadoop-hdds/docs/content/integration/DistCp.md

```diff
@@ -87,3 +87,78 @@ By specifying the appropriate checksum configuration or skipping the validation,
 When data resides in an HDFS encryption zone or Ozone encrypted buckets, the file checksum will not match. This is because the underlying block data differs due to the use of a new EDEK (Encryption Data Encryption Key) at the destination. In such cases, specify the `-skipcrccheck` parameter to avoid job failures.
 
 For more information about using Hadoop DistCp, consult the [DistCp Guide](https://hadoop.apache.org/docs/current/hadoop-distcp/DistCp.html).
+
+## Troubleshooting Common Issues
+
+### Delegation Token Issues
+If a DistCp command fails and the error output contains "OzoneToken", indicating an issue with retrieving a delegation token from the destination (or source) Ozone cluster, ensure that Ozone's security is explicitly enabled in the client's Hadoop configuration.
+
+Add the following property to `core-site.xml` or `ozone-site.xml` on the node where you run the DistCp command:
```

Comment:
`ozone.security.enabled` is an Ozone configuration, should not be added in `core-site.xml`.

```suggestion
Add the following property to `ozone-site.xml` on the node where you run the DistCp command:
```